roles/pulibrary.ruby/tasks/main.yml

```diff
@@ -3,29 +3,25 @@
   apt:
     name: software-properties-common
     state: present
-    cache_valid_time: '{{ apt_cache_timeout }}'
-    update_cache: 'yes'
+    update_cache: true
 
 - name: add the Brightbox repository
   apt_repository:
-    repo: 'ppa:brightbox/ruby-ng'
+    repo: "ppa:brightbox/ruby-ng"
     state: present
-    update_cache: 'yes'
+    update_cache: true
 
 - name: install Ruby 2.4.3
```

Comment:
It isn't scoped for this PR, but please be aware that Brightbox now supports 2.6 releases: https://www.brightbox.com/docs/ruby/ubuntu/